_wiki/Install/Linux.md

```diff
@@ -9,9 +9,7 @@ title: Linux Installation Guide
 > This assumes that you are in the directory in which you downloaded OpenTabletDriver to.
 
 ```sh
-# Install the .NET runtime
 sudo apt update
-sudo apt install -y dotnet-sdk-6.0
```

Comment:
does it auto pull now? (I just want it noted in some way)

Reply:
Yep, the correct dependencies will be pulled automatically. Less stuff in wiki for us to keep track of.